README.md

```diff
@@ -1,2 +1,9 @@
-# redis
-Official Layer0 Redis Service
+# Layer0 Redis
```

Comment:
Can we make this (and all future) layer1 repos match https://github.com/quintilesims/consul?

```
/
README.md
example/
terraform/
```